datalad/distribution/tests/test_update.py

```diff
@@ -79,6 +79,10 @@ def test_update_simple(origin, src_path, dst_path):
     # forget we cloned it by removing remote, which should lead to
     # setting tracking branch to target:
     source.repo.remove_remote(DEFAULT_REMOTE)
+    # also forget the declared absolute location of the submodules, and turn them
+    # relative to this/a clone
+    for sub in source.subdatasets(result_xfm=lambda x: x['gitmodule_name']):
+        source.subdatasets(path=sub, set_property=[('url', './{}'.format(sub))])
```

Comment:
same here -- a new scenario or needed (i.e. something what worked before, no longer works)?

Reply:
and same answer as above. The very nature of this PR makes this change necessary.